src/docs/guides/variables.md

```diff
@@ -96,9 +105,29 @@ alt="Screenshot of Variables Pane"
 layout="responsive"
 width={2408} height={1150} quality={100} />
 
+## Sealed Variables
+
+Sealed variables are useful when you want to provide a value to your builds and deployments but not have it be visible in the UI (or via the API).
+
+### Sealing a variable
+
+To seal a variable, click the 3-dot menu on the right-side of an existing variable and click the `Seal` button.
+
+### Caveats
```

Comment:
mention they cannot be updated via the Raw Editor?  or like Brody mentioned, add something about how to edit them.  could be after "Sealing a variable", "Updating a sealed variable"